indexer/search_string_utils.cpp

```diff
@@ -432,6 +428,44 @@ class StreetsDirectionsHolder
   }
 };
 
+class StreetsDirectionsHolder : public SynonymsHolderBase
+{
+public:
+  StreetsDirectionsHolder()
+  {
+    // ("short name", "full name")
+    for (auto const * s : {"n", "north", "s", "south", "w", "west", "e", "east",
```

Comment:
These shortcuts are very specific to countries and languages. E.g. they should not be applied in German or French-speaking countries, for example. Is there any plan to take it into account? Any specific issue?

Reply:
FYI @vng here are some other similar lists that may help complete this for the US

https://wiki.openstreetmap.org/wiki/Seattle_Import#SeattleAddress_Transformation

https://git.zyphon.com/will/the-villages-import/src/branch/main/qgis-functions.py#L32